recipe/meta.yaml

```diff
@@ -19,11 +19,13 @@ package:
 source:
   url: https://github.com/single-cell-data/TileDB-SOMA/archive/{{ version }}.tar.gz
   sha256: {{ sha256 }}
+  patches:   # [osx and not arm64]
+    - 0001-version-py-workaround-1-11-2.patch
 
```

Comment:
A few notes:

1. I think we should apply the patch to all the builds. We know code will be in the next release, so we may as well test it now
2. I don't think the Jinja2 preprocessing selectors are that smart. I think you have to apply it to each line you want to remove
3. The `version.py` issues are with the [linux-64 builds](https://github.com/TileDB-Inc/tiledbsoma-feedstock/pull/157#issuecomment-2124984583), so if we are going to selectively apply the patch, I think we would use `# [linux]`

Reply:
> I think we should apply the patch to all the builds. We know code will be in the next release, so we may as well test it now

Sorry, I lack your expersite -- how am I not? What am I doing wrong?

> I don't think the Jinja2 preprocessing selectors are that smart

Here too -- where is Jinja involved here?

Reply:
> we would use # [linux] 

This one I think I understand -- mindless copypasta from my recent work on `r-tiledb-feedstock` -- I'll update ...

Reply:
https://github.com/TileDB-Inc/tiledbsoma-feedstock/pull/157/commits/c51fec9ba6e927cec508adcd892ae6515f7a3475
🤞

Reply:
> Here too -- where is Jinja involved here?

The preprocessing selectors like `[linux]` and `[osx and not arm64]` are processed by the Python package jinja2. If the variable is true, the line is kept. If the variable is false, the line is removed from the recipe.

My first suggestion was to remove all of the selectors from the patch, so that the patch would be applied to all of the builds. You implemented my other suggestion, which was to use `# [linux]`. This is fine since it applies the patch only to the linux builds, which is what we need for green CI.

Reply:
OK thanks @jdblischak ... as always I creep slowly in the direction of understanding when it comes to Conda ... 🤞